### PR TITLE
🎨 wrap missing strings in portal

### PR DESCRIPTION
**Pull request by cathysarisky**
Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [X ] There's a clear use-case for this code change, explained below
- [X] Commit message has a short title & references relevant issues
- [X] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

Added sneaky missing translation wrapping, notably the elusive /year and /month strings. Also "Change Plan" and "Choose Plan" for upgrade situations.

I opted to include the slash in /year and /month , to give translators more options for interval presentation.

### CODE REVIEW

File: apps/portal/src/components/common/ProductsSection.js [hunk 1]
```diff
@@ -594,12 +594,13 @@ function ProductCardTrialDays({trialDays, discount, selectedInterval}) {
...
 function ProductCardPrice({product}) {
     const {selectedInterval} = useContext(ProductsContext);
-    const {site} = useContext(AppContext);
+    const {t, site} = useContext(AppContext);
     const monthlyPrice = product.monthlyPrice;
     const yearlyPrice = product.yearlyPrice;
     const trialDays = product.trial_days;
     const activePrice = selectedInterval === 'month' ? monthlyPrice : yearlyPrice;
     const alternatePrice = selectedInterval === 'month' ? yearlyPrice : monthlyPrice;
+    const interval = activePrice.interval === 'year' ? t('/year') : t('/month');
```
Comment (sagzy) [inline apps/portal/src/components/common/ProductsSection.js:603]: Could we use `t('year')` and `t('month')` in the translation strings (without the `/`)?

That way, we'd be able to re-use the same translations in more situations

**Review by sagzy — COMMENTED** [review]
Hey @cathysarisky,

Thanks a lot for this PR.

I've added one improvement suggestion, LGTM otherwise!

**Comment by ErisDS** [comment]
Hey @cathysarisky thank you so so much for getting this done 🙏 . Would you mind making the change @sagzy suggested? Other than that this is good to go! 🎉

**Comment by cathysarisky** [comment]
I'm wondering if I needed to bump the version of portal?  I don't see the changes in a fresh Ghost install today... 
(Sorry if I missed a step!)

**Comment by sagzy** [comment]
> I'm wondering if I needed to bump the version of portal? I don't see the changes in a fresh Ghost install today... (Sorry if I missed a step!)

Hey @cathysarisky, you're spot on that we need to bump Portal when we add new i18n strings to it. But our bad, not yours! :) I have now released a patch version of Portal (v2.43.2), which contains the new i18n strings.

**Pull request merged**